crates/rethnet_rpc_server/src/lib.rs

```diff
@@ -980,42 +980,37 @@ impl Server {
         event!(Level::INFO, "Listening on {}", config.address);
 
         let (local_accounts, genesis_accounts): (
-            HashMap<Address, SecretKey>,
+            HashMap<Address, k256::SecretKey>,
             HashMap<Address, AccountInfo>,
         ) = {
-            let secp256k1 = Secp256k1::signing_only();
             config
                 .accounts
                 .iter()
                 .enumerate()
-                .map(
-                    |(
-                        i,
-                        AccountConfig {
-                            private_key,
-                            balance,
+                .map(|(i, account_config)| {
+                    let AccountConfig {
+                        secret_key,
+                        balance,
+                    } = account_config;
+                    let address = public_key_to_address(secret_key.public_key());
+                    event!(Level::INFO, "Account #{}: {address:?}", i + 1);
+                    event!(
+                        Level::INFO,
+                        "Private Key: 0x{}",
```

Comment:
```suggestion
                        "Secret Key: 0x{}",
```

Reply:
Done in [e3ec175](https://github.com/NomicFoundation/hardhat/pull/4465/commits/e3ec175495c36b67645a5e9aba0d3151eb910729)